src/Datadog.Trace/Agent/ZipkinSerializer.cs

```diff
@@ -16,15 +16,15 @@ internal class ZipkinSerializer
         // Don't serialize with BOM
         private readonly Encoding utf8 = new UTF8Encoding(false);
 
-        public void Serialize(Stream stream, Span[][] traces, TracerSettings settings)
```

Comment:
Good removing this. This was a left-over from using some setting to set headers when doing Zipkin that was later removed but the `settings` were still being passed.